src/features/projectsV2/ProjectsContext.tsx

```diff
@@ -73,18 +83,27 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
   const [plantLocations, setPlantLocations] = useState<PlantLocation[] | null>(
     null
   );
+  const [selectedPlantLocation, setSelectedPlantLocation] =
+    useState<PlantLocation | null>(null);
+  const [samplePlantLocation, setSamplePlantLocation] =
+    useState<SamplePlantLocation | null>(null);
+  const [hoveredPlantLocation, setHoveredPlantLocation] = useState<
+    PlantLocation | SamplePlantLocation | null
+  >(null);
   const [selectedClassification, setSelectedClassification] = useState<
     TreeProjectClassification[]
   >([]);
   const [debouncedSearchValue, setDebouncedSearchValue] = useState('');
   const [isSearching, setIsSearching] = useState(false);
-  const [selectedSite, setSelectedSite] = useState(0);
+  const [selectedSite, setSelectedSite] = useState<number | null>(null);
```

Comment:
I suspect this change has resulted in not selecting a site initially when I directly go the the project details page link.